src/Components/FloatingButton.lua

```diff
@@ -66,6 +67,31 @@ function FloatingButton:render()
 					Position = UDim2.fromScale(0.5, 0.5),
 					Size = UDim2.new(props.ImageSize, props.ImageSize),
 				}),
+
+				ToolTip = e("TextLabel", {
+					Text = props.ToolTipText,
+					TextColor3 = theme:GetColor("BrightText", "Default"),
+					BackgroundTransparency = 0,
+					Position = UDim2.fromScale(1, -0.5),
+					AnchorPoint = Vector2.new(1, 0.5),
+					AutomaticSize = Enum.AutomaticSize.XY,
+					BackgroundColor3 = theme:GetColor("ScrollBarBackground", "Default"),
+					Visible = self.hovered:map(function(hovered)
+						return hovered
+					end),
```

Comment:
Instead of this, make hovered a state and render tooltip with `self.state.hovered and e("TextLabel", {`

Reply:
Yeah originally hovered was already a binding, so just used it as such; I'll do what I was going to originally and use state for these values.